features/IFC102_Absence-of-deprecated-entities.feature

```diff
@@ -1,74 +1,337 @@
+      | IfcWallType                     | "POLYGONAL" or "STANDARD"      |
+
+
+  Scenario Outline: Check for deprecated enumerated values - IFC4
+    # IfcStructuralLoadGroup: IfcLoadGroupTypeEnum
+    # IfcWall: IfcWallTypeEnum
+
+    Given A model with Schema "IFC4"
+    Given an <Entity>
+
+    Then PredefinedType is not <Value>
+
+    Examples:
+      | Entity                   |  Value                                       |
+      | IfcStructuralLoadGroup   | "LOAD_COMBINATION"                           | 
+      | IfcWall                  | "POLYGONAL" or "STANDARD" or "ELEMENTEDWALL" |
+      | IfcWallType              | "POLYGONAL" or "STANDARD" or "ELEMENTEDWALL" |
+
+
+  Scenario Outline: Check for deprecated enumerated values - IFC2X3
+    # IfcOwnerHistory: IfcChangeActionEnum
+
+    Given A model with Schema "IFC2X3"
+    Given an <Entity>
+
+    Then <Attribute> is not <Value>
+
+    Examples:
+      | Entity                   | Attribute            |         Value                                |
+      | IfcOwnerHistory          | ChangeAction         | "MODIFIEDADDED" or "MODIFIEDDELETED"         |
+
+
+  Scenario Outline: Check for deprecated explicitly instantiated entities - IFC4.3
+
+    Given A model with Schema "IFC4X3"
+    Given an <Entity>
+
+    Then its type is not <Entity> excluding subtypes
+
+    Examples:
+      | Entity                     | 
+      | IfcFlowFitting             |
+      | IfcFlowSegment             | 
+      | IfcFlowTerminal            | 
+      | IfcFlowController          | 
+      | IfcFlowMovingDevice        | 
+      | IfcFlowStorageDevice       | 
+      | IfcFlowTreatmentDevice     | 
+      | IfcEnergyConversionDevice  |
+
+
+  Scenario Outline: Check for deprecated explicitly instantiated entities - IFC4
+
+    Given A model with Schema "IFC4"
+    Given an <Entity>
+
+    Then its type is not <Entity> excluding subtypes
+
+    Examples:
+      | Entity                     | 
+      | IfcFlowFitting             |
+      | IfcFlowSegment             | 
+      | IfcFlowTerminal            | 
+      | IfcFlowController          | 
+      | IfcFlowMovingDevice        | 
+      | IfcFlowStorageDevice       | 
+      | IfcFlowTreatmentDevice     | 
+      | IfcEnergyConversionDevice  |
+
+
+  Scenario Outline: Check for deprecated explicitly instantiated entities - IFC2X3
+
+    Given A model with Schema "IFC2X3"
+    Given an <Entity>
+
+    Then its type is not <Entity> excluding subtypes
+
+    Examples:
+      | Entity                     | 
+      | IfcProductRepresentation   |
+      | IfcRepresentation          | 
+      | IfcRepresentationContext   | 
+      | IfcRelAssociates           | 
+
+
+  Scenario: Check for deprecated attribute values - IFC4.3
+
+    Given A model with Schema "IFC4.3"
+    Given an IfcShapeRepresentation
+
+    Then RepresentationType is not "PointCloud"
+
+
+  Scenario: Check for deprecated property set - IFC4.3
```

Comment:
Similar to a comment above. Isn't enough the write this for IFC2X3, and then it applies to IFC4 and IFC4.3 too? It's the same Pset, isn't it?